.github/workflows/elixir.yml

```
@@ -18,11 +18,14 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: '1.11'
-              otp: '21.3'
+              elixir: '1.14'
+              otp: '26.0'
```

The minimum supported OTP version is 23.0 for Elixir v1.14. The strategy here was to test with the oldest and newest versions.

```suggestion
              otp: '23.3'
```